docs/platform/howto/custom-plans.rst

```diff
@@ -0,0 +1,48 @@
+Requesting custom plans
+=======================
+
+Our stock plans are designed to cover most common use cases. The plans
+are tested and optimized configurations that work well in different
+clouds.
+
+Sometimes, especially for very high throughput clusters or other special
+use cases, the stock plans might not be an optimal fit. In such cases we
+are happy to look into custom plans where the service configurations are
+changed to best fit your needs.
+
+Custom plans can be made available for plans starting from 500
+USD/month.
+
+Some variables which may be adjusted in custom plans include:
+
+-  Amount of storage
+
+-  Amount and frequency of backups
+
+-  Amount of nodes
+
+-  CPU/RAM configuration per node
+
+The possible configurations depend on several factors, such as the
+cloud, region, available instance types and the service type in
+question.
+
+For example available memory and CPU configurations may depend on
+restrictions imposed by the cloud provider and the instance types
+available in the region.
+
+Please feel free to contact us via Intercom or sales@aiven.io for a
+quote and the availability of a custom plan.
+
+For the custom plan request, please provide:
+
+-  The **region** and **cloud** you would like it in (e.g. "Google
+   cloud, us-east1")
+
+-  The **service type** (e.g. "Kafka" or "PostgreSQL")
```

Comment:
This is pretty tricky, @TibsAtWork what's a good way to handle these service types, they're not code/literal values, but we also haven't had a reason to mention the product names with trademark earlier in the document. Any thoughts?